src/Modules/Pages/Domain/Revision/Form/RevisionType.php

```diff
@@ -65,25 +65,23 @@ function (FormEvent $event): void {
                         'disable_allow_children' => $revisionDTO->page->isForbiddenToHaveChildren(),
                     ]
                 );
-                $tabs->get(TabsType::getTabNameForLabel('lbl.SEO'))->add(
-                    'meta',
-                    MetaType::class,
-                    [
-                        'disable_slug_overwrite' => $revisionDTO->page->isHome(),
-                        'base_field_name' => 'title',
-                        'base_url' => $this->pageRouter->getRouteForPageId(
-                            $revisionDTO->parentPage !== null
-                                ? $revisionDTO->parentPage->getId()
-                                : Page::PAGE_ID_HOME
-                        ),
-                        'generate_slug_callback_class' => RevisionRepository::class,
-                        'generate_slug_callback_method' => 'generateSlug',
-                        'generate_slug_callback_parameters' => [
-                            $revisionDTO->locale,
-                            $revisionDTO->hasEntity() ? $revisionDTO->getEntity()->getId() : null,
-                        ],
-                    ]
-                );
+                $hasEntity = $revisionDTO->hasEntity();
```

Comment:
This code has been updated and works different now, no longer with the MD5, can you update your style fix?